cbor-schema/plt.cddl

```diff
@@ -11,18 +11,24 @@ token-holder-transaction = [ * token-holder-operation ]
 ; the type of the operation.
 token-holder-operation = token-transfer
 
+; A token holder event, which is the result of executing a token holder operation.
+token-holder-event = token-transfer-details
+
 ; A token transfer operation. This transfers a specified amount of tokens from the sender account
 ; (implicit) to the recipient account.
 token-transfer = {
     ; The operation type is "transfer".
-    "transfer": {
-        ; The amount of tokens to transfer.
-        "amount": token-amount,
-        ; The recipient account.
-        "recipient": token-receiver,
-        ; An optional memo.
-        ? "memo": memo
-    }
+    "transfer": token-transfer-details
+}
+
+; The details of a token transfer operation.
+token-transfer-details = {
+    ; the amount of tokens to transfer.
+    "amount": token-amount,
+    ; the recipient account.
+    "recipient": token-receiver,
```

Comment:
I guess the sender is always clear from context?

Reply:
For the transfer operation, the sender is implicit. However, for the transfer event, I think it should be explicit.

Reply:
Why do you think it should be explicitly defined at this level? It's already available from the surrounding structure, is it not? When looking at the corresponding "simple transfer" event, it does not explicitly define the sender either, but rather rely on the surrounding context for this.

Reply:
When we get sponsored transactions and smart contract interaction, the sender might be harder to derive from the context.